### DISCUSSION TITLE
Passwort reset not working

### DISCUSSION THREAD

**Discussion by Seegee06** [mastodon/mastodon#22501] [Q&A] [answered]
I just started some weeks ago with mastodon mstdn.social.
However, I wanted to login on another NB but I'm getting always the error message: wrong eMail or Password.
So I tried to reset my PW but in vain.
In the end it seems that my eMail (even the fact I'm still registered and online) is not registered at Mastodon.
I have no desire to lose my followers who already follow me on Mastodon again.

Any suggestions Mastodon support ?

**Comment by nightpool** [comment, marked as answer]
> However, I wanted to login on another NB

What does this mean? What is an "NB"?

If you created your account on mstdn.social, you need to log into your account on mstdn.social. Just like if you have an account on Gmail, you can't use that password to log into Yahoo.

**Comment by nightpool** [comment]
This forum is for the development and maintenance of the Mastodon server software. This is a free software package that can be run by anyone who wants to provide their own, connected, federated social media software. It is not a website, just code that others use to run a website.

The people who maintain this software are, for the most part, not involved or associated with the individual websites that people run or sign up on. You'll have to contact the person running your server to get more details about what may or may not be happening here. We cannot provide support for individual server issues, since we have no connection to the individual server admins.

**Comment by Seegee06** [comment]
NB = Notebook

I clearly missed to explain my problem.

I'm logged in on mstdn.social with my first Notebook.
Now I want to login with my Smartphone and second Notebook as well but I
can't since it say's that my eMail doesn't exist.
Anyway, the status of my account on the first Notebook is "fully
operational".
Best

Am Mo., 19. Dez. 2022 um 17:18 Uhr schrieb nightpool <
***@***.***>:

**Comment by Seegee06** [comment]
Problem solved
